scripting/shavit-timelimit.sp

```diff
@@ -244,6 +275,50 @@ void SetLimit(int time)
 	gCV_RestartGame.IntValue = 1;
 }
 
+public Action CheckRemainingTime(Handle timer)
+{
+	int timelimit;
+
+	if(!GetMapTimeLimit(timelimit) || timelimit == 0)
+	{
+		return Plugin_Continue;
+	}
+
+	if(timelimit > 0)
```

Comment:
Isn't this part sort of pointless? You can't have a negative timelimit, and you check if the timelimit is zero above?